migrations/versions/3b80e4ac0ecb_.py

```diff
@@ -0,0 +1,31 @@
+"""empty message
+
+Revision ID: 3b80e4ac0ecb
+Revises: e3caa0f2a16c
+Create Date: 2019-02-05 08:04:37.035686
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '3b80e4ac0ecb'
+down_revision = 'e3caa0f2a16c'
```

Comment:
```suggestion
down_revision = ('e3caa0f2a16c', '91ee86a38001')
```

Try this and delete the empty migration.